core/src/poh_recorder.rs

```diff
@@ -27,17 +26,28 @@ use std::cmp;
 use std::sync::mpsc::{channel, Receiver, SendError, Sender, SyncSender};
 use std::sync::{Arc, Mutex};
 use std::time::Instant;
+use thiserror::Error;
 
 const GRACE_TICKS_FACTOR: u64 = 2;
 const MAX_GRACE_SLOTS: u64 = 3;
 
-#[derive(Debug, PartialEq, Eq, Clone)]
+#[derive(Error, Debug, Clone)]
 pub enum PohRecorderError {
+    #[error("invalid calling object")]
     InvalidCallingObject,
+
+    #[error("max height reached")]
     MaxHeightReached,
+
+    #[error("max height not reached")]
```

Comment:
```suggestion
    #[error("min height not reached")]
```

Reply:
Thanks! Fixed in 81d1e85c95b6187ecd50c849fb19a365dd2d1b28